src/utils/documentUtils.ts

```diff
@@ -40,4 +41,36 @@ const getDocPropsProperties = async (zip: JSZip): Promise<{ doc: Document; prope
     return { doc, properties };
 };
 
-export default { createOrUpdateProperty, getDocPropsProperties };
+const getCellReference = (col: number, row: number): string => {
+    // 65 is the ascii value of first column 'A'
+    return String.fromCharCode(col + 65) + "$" + row.toString();
+};
+
+const getTableReference = (numberOfCols: number, numberOfRows: number) => {
+    return `A1:${getCellReference(numberOfCols, numberOfRows)}`.replace("$", "");
+};
+
+const createCell = (doc: Document, colIndex: number, rowIndex: number, dataType: number, data: string) => {
```

Comment:
maybe cellContent is a better name here?

Reply:
now that I see you send to updateCellData a data and a cellData I think maybe I don't understand what is data refers to here..

Reply:
It is an xml Element, represents the cell. I'll change the name to createCellElement